docs/start/prereq.md

```diff
@@ -1,13 +1,31 @@
-# Prerequisites for using PolkaBTC
+# Getting started with PolkaBTC
+
+
+To start using PolkaBTC Beta, follow the steps below to set up a Bitcoin and Polkadot testnet wallet. 
 
 ### 1. Installing a Bitcoin testnet wallet
 
 You will need a Bitcoin testnet wallet to test PolkaBTC.
 For a general overview of Bitcoin wallets you can consult [bitcoin.org's wallet selector](https://bitcoin.org/en/choose-your-wallet?step=5).
 
+
+You can also pick one of the following Bitcoin testnet wallets below:
+
+- **GreenAddress**: https://test.greenaddress.it/en/ (Android and IOS)
+- **Bitcoin Testnet Wallet**: https://play.google.com/store/apps/details?id=de.schildbach.wallet_test (Android only)
+- **Electrum**: https://electrum.org/#home (Linux, Windows, macOS and Android)
+- **Bitpay**: https://bitpay.com/wallet/ (Linux; Guide here: https://support.bitpay.com/hc/en-us/articles/360015463612-How-to-Create-a-Testnet-Wallet)
+
+#### Hardware Wallets
+You can also use your hardware wallet if it has Bitcoin testnet support.
```

Comment:
Nitpick: "You can also use your hardware wallet. The following have Bitcoin testnet support."